scripts/screeps_rl_experiment_card.py

```
@@ -2552,6 +2576,13 @@ def validate_simulation(raw: Any) -> None:
     for field in ("ticks", "workers", "repetitions"):
         if positive_int(raw.get(field)) is None:
             raise CardValidationError(f"simulation.{field} must be a positive integer")
+    for field, aliases in (
+        ("scale_environments", ("scale_environments", "scaleEnvironments")),
+        ("min_concurrent_environments", ("min_concurrent_environments", "minConcurrentEnvironments")),
+    ):
+        value = first_present(raw, aliases)
+        if value is not None and positive_int(value) is None:
+            raise CardValidationError(f"simulation.{field} must be a positive integer")
```

Ruff (0.15.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)